lazycatlabs/auth_api
# refactor: user feature

## What it does

- refactor the user feature
- remove unused files and code

## How to test

1. clone the project and start it
2. test all API related to user
3. observe the view
4. there should be no issues

## Screenshot
n/a
